_posts/2022-09-20-discourse

```diff
@@ -0,0 +1,28 @@
+---
+  layout: post
+  author: trnstlntk
+  title: "From mailing lists and Gitter to a web forum: welcoming your feedback"
+  category: Blog
+
+---
+<img src="https://openrefine.org/images/2022-jan-ainali.jpg" alt="Simple graphic showing group discussions with multicolored text balloons" class="inset" />
```

Comment:
The filename for the image is different from the one you added :)

Reply:
Maybe Jan's happy face would be a great illustration! 🙈

Reply:
I replaced it with the one I intended to use. Was just to test if you were paying attention 🤪